config/config.go

```diff
@@ -314,10 +316,8 @@ func (c *Config) extend(extensionConfig Config) {
 			}
 			// The keywords from the base rule and the extended rule must be merged into the global keywords list
 			for _, k := range baseRule.Keywords {
```

Comment:
This code is covered by https://github.com/gitleaks/gitleaks/pull/1633/files#diff-373f5578b8f362a364b50b0d01919a57da64c128547ca10d49df8ae422237c82R405